articles/azure-arc/vmware-vsphere/recover-from-resource-bridge-deletion.md

```diff
@@ -11,23 +11,17 @@ ms.author: v-gjeronika
 manager: jsuri
 ---
 
-# Recover from accidental deletion of resource bridge VM
+# Recover from accidental deletion of resource bridge VM or it's irrecoverable
```

Comment:
This is not correct English. What do we want to convey?

Reply:
@riva-yadav please follow up with Sayan to update and close this PR. He is the one that created this.

Reply:
Wanted to convey the fact that Disaster Recovery can be done not only for accidental VM deletion but also for various other reasons. I have removed the Header changes, since the intent is captured in this statement already:
> In this article, you learn how to recover the Azure Arc resource bridge connection into a working state in disaster scenarios such as accidental deletion.

Examples of other scenarios when a DR can be performed:
    1. The IP addresses for the network was changed (and appliance became Offline).
    2. The Proxy configuration was changed (and appliance became Offline).
    3. There was change in the resource pool, network, folder, datastore, datacenter (either the entity's full path changed or the VM got migrated to some other entity).